src/main/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+abstract class Machine {
```

Comment:
Consider using access modifiers for the class declaration to follow the encapsulation principle. For example, you might want to make this class public if it is intended to be accessed from outside its package.